test/fixture/snapshots/test-sourcemaps/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scripts": {
+	"build": "tsc"
```

Comment:
Does this assume `tsc` is installed globally? I know we have a dev dependency on it but running `build` inside this directory won't pick it up. This should either refer to the top-level `node_modules/.bin/tsc` or use `npx tsc`.

(Also, going by the GitHub output here, the indentation seems off in this file and `tsconfig.json`.)